app/models/course.rb

```diff
@@ -44,7 +44,7 @@ class Course < ActiveRecord::Base
   # @!method with_owners
   #   Includes all course_users with the role of owner.
   scope :with_owners, (lambda do
-    course_users = CourseUser.owner.where(course: pluck(:id))
+    course_users = CourseUser.owner.with_approved_state.where(course: pluck(:id)).includes(:user)
```

Comment:
In Global course management,  and it also requires user to be loaded. I think no other place will need this, because normally only index action need eager loading I think ?

And since we are displaying user instead of course_user across the app, this is kind of make sense IMO. What do you think ?